roofit/roofitcore/inc/RooFFTConvPdf.h

```diff
@@ -107,7 +107,7 @@ class RooFFTConvPdf : public RooAbsCachedPdf {
   virtual PdfCacheElem* createCache(const RooArgSet* nset) const ;
   virtual TString histNameSuffix() const ;
 
-  // mutable std::map<const RooHistPdf*,CacheAuxInfo*> _cacheAuxInfo ; //! Auxilary Cache information (do not persist)
+  // mutable std:: map<const RooHistPdf*,CacheAuxInfo*> _cacheAuxInfo ; //! Auxilary Cache information (do not persist)
```

Comment:
That was probably not intended, but I guess it's OK to leave it in for now.

Reply:
It was partially intended - I just want to suppress output from my script that `std::map` used without including. My small script does not recognize if class placed in comments